test/JDBC/input/datatypes/TestVariableDataLength-vu-prepare.txt

```diff
@@ -0,0 +1,42 @@
+# all variable length datatypes like numeric, decimal, time, datetime2, datetimeoffset need explicit checking
```

Comment:
this change has both boundary and negative test cases. Let us update the description with relevant test details

Reply:
Done